crates/core/src/models/felt.rs

```diff
@@ -90,3 +90,106 @@ impl From<Felt252Wrapper> for U256 {
         U256::from_be_bytes(felt.to_bytes_be())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use hex::FromHex;
+
+    use super::*;
+
+    const MAX_ADDRESS: &str = "ffffffffffffffffffffffffffffffffffffffff";
```

Comment:
can we use `2 ** 160 - 1` and `2 ** 160` instead? easier to read, and `address<>uint160` in solidity